config/src/main/java/com/epam/aidial/core/config/AccessControlled.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.aidial.core.config;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import lombok.Data;
+
+import java.util.Collection;
+import java.util.Set;
+
+@Data
+public abstract class AccessControlled {
+
+    private String name;
+
+    @JsonAlias({"userRoles", "user_roles"})
+    private Set<String> userRoles;
+
+    public boolean hasAccess(Collection<String> roles) {
```

Comment:
The method confuses a bit. 

Perhaps we need to have better naming for variables. Actually we want to match expected to actual user roles. It's not clear what roles are expected/required and what - actual.

Reply:
Please add javadoc

Reply:
added javadoc